README.md

```diff
@@ -1,39 +1,63 @@
-<!-- 
-This README describes the package. If you publish this package to pub.dev,
-this README's contents appear on the landing page for your package.
+# Help Ukraine Widget for Flutter
+This is a port of [Help Ukraine Widget](https://helpukrainewinwidget.org/) -
+this widget helps to let users of your app know how they can help Ukraine
+from anywhere in the world.
 
-For information about how to write a good package README, see the guide for
-[writing package pages](https://dart.dev/guides/libraries/writing-package-pages). 
+### Some examples
 
-For general information about developing packages, see the Dart guide for
-[creating packages](https://dart.dev/guides/libraries/create-library-packages)
-and the Flutter guide for
-[developing packages and plugins](https://flutter.dev/developing-packages). 
--->
+<details open><summary>First widget variation</summary>
 
-TODO: Put a short description of the package here that helps potential users
-know whether this package might be useful for them.
+![](gifs/widget1.gif)
+</details>
+
+<details><summary>Second widget variation</summary>
+    
+![](gifs/widget2.gif)
+</details>
+
+<details><summary>Third widget variation</summary>
+
+![](gifs/widget3.gif)
+</details>
+
+<details><summary>Fourth widget variation</summary>
+
+![](gifs/widget4.gif)
+</details>
 
 ## Features
+By installing this widget in your app, you will help to end the war in Ukraine.
+It will provide links for:
 
-TODO: List what your package can do. Maybe include images, gifs, or videos.
+* [donations](https://uahelp.monobank.ua/),
+* sharing this widget
+* and [other ways to support Ukraine](https://war.ukraine.ua/).
 
 ## Getting started
+<h5 a><strong><code>pubspec.yaml</code></strong></h5>
```

Comment:
HTML here is uncalled for, let's stick to plain Markdown wherever possible.
Besides, we could be a bit more verbose:
```suggestion
Add the package as a dependency to your __pubspec.yaml__:
```